iclr2016(old scripts)/20160114-demo/web-field-workshop.html

```diff
@@ -1,8 +1,4 @@
 <html>
-  <head>
-    <!--<script src="https://code.jquery.com/jquery-1.10.1.min.js"></script>-->
-    <script src="https://cdn.jsdelivr.net/lodash/3.5.0/lodash.compat.min.js"></script>
```

Comment:
another naive question: I understand that removing this code stops the file from loading the older version of lodash, but where does the newer version come from, now that this is gone?